GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/short-heading.config

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like a temporary test page that's crept into the PR?

Reply:
This file has been removed